src/client/components/AddResume/AddResume.js

```diff
@@ -22,20 +23,24 @@ export const AddResume = () => {
   const handleChange = (e) => {
     const selectedFile = e.target.files[0];
 
-    setUploadedFile(selectedFile);
+    setUploadedMyFile(selectedFile);
 
     if (selectedFile) {
       if (types.includes(selectedFile.type)) {
         setError(null);
-        setUploadedFile(selectedFile);
+        setUploadedMyFile(selectedFile);
       } else {
-        setUploadedFile(null);
+        setUploadedMyFile(null);
         setError('Please select a pdf file');
       }
     }
   };
 
-  useStorage(uploadedFile);
+  const {url} = useStorage(uploadedMyFile);
+
+  const clickHandler = () => {    
+        PostUrlComponent({uploadedFile:url, title, description})
+      };
```

Comment:
@pdsorensen @hamedveloper both component work properly. but when data send by the clickHandler we are getting this error: 
![image](https://user-images.githubusercontent.com/19622435/108894652-d6ca4080-7612-11eb-95a9-f0fef854a589.png)